packages/orbit-components/src/utils/rtl/README.md

```diff
@@ -19,3 +19,30 @@ In addition, you also need to set the `dir` property on the `html` element to ha
   ...
 </html>
 ```
+
+## Rendering Bidirectional Text in RTL Mode
+
+When rendering bidirectional text (mixing both LTR and RTL characters) in RTL mode, it's important to understand how the browser handles the rendering.
+
+### Fully RTL Text
+
+If the text is fully RTL, such as numbers and Hebrew characters, it will be rendered properly.
+
+### Bidirectional Text
+
+However, when using bidirectional text, such as "2 of 6", the rendering may not be as expected. The browser considers it as RTL due to the `dir="rtl"` attribute and starts rendering from right to left.
+
+The browser begins with "2", which can be both LTR and RTL. When it encounters the "of 6" part, which is considered LTR, it results in the rendering of "of 6 2".
+
+### Handling Bidirectional Text
+
+To handle bidirectional text correctly:
+
+1. Use fully RTL text to ensure proper rendering.
+2. If using bidirectional text is necessary, wrap the LTR part with an element that has `dir="ltr"` to enforce the correct rendering order.
+
+```html
+<span dir="rtl">שלום <span dir="ltr">world!</span></span>
```

Comment:
```
<div dir="rtl">
<span dir="rtl">שלום <span dir="ltr">world!</span></span>
</div>
```
puts the exclamation mark in the wrong place, so it probably should be
```
<div dir="rtl">
<span dir="rtl">שלום <span dir="ltr">world</span>!</span>
</div>
```
to be more correct?

Also, what about the `<bdi>` and `<bdo>` tags? They might be more idiomatic than `<span dir="...">`. WDYT?

Reply:
Re. exclamation mark – not sure what’s correct from the linguistic perspective. Maybe it would be better to remove the mark altogether.

I think `bdo` is for quite specific use cases (as noted in https://www.w3.org/International/articles/inline-bidi-markup/) but `bdi` is a good point. It’s also recommended in the latter doc for some cases. But for the cases when the directionality is known they recommend to use a span with dir attr without any further reasoining 🤔

Reply:
I added a third bullet point explaining the use case of `bdi` as it seems relevant.